src/app/components/UserFeedback.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import Button from 'material-ui-next/Button';
+import Card, { CardContent } from 'material-ui-next/Card';
+import IconButton from 'material-ui-next/IconButton';
+import Tooltip from 'material-ui-next/Tooltip';
+// import Dismiss from 'material-ui/svg-icons/content/clear';
+import FaceFrown from '../../assets/images/feedback/face-frown';
+import FaceNeutral from '../../assets/images/feedback/face-neutral';
+import FaceSmile from '../../assets/images/feedback/face-smile';
```

Comment:
Cool way to import and use SVG in the app. Not sure about the path they should live in.

Reply:
that path sounds good